source/common/runtime/runtime_features.cc

```diff
@@ -134,6 +134,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_enable_universal_header_validator)
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_defer_logging_to_ack_listener);
 // TODO(#31276): flip this to true after some test time.
 FALSE_RUNTIME_GUARD(envoy_restart_features_use_fast_protobuf_hash);
+// TODO(panting): flip this to true step by step
```

Comment:
nit: What does "step by step" mean in this context?

Reply:
I mean roll out gradually, made the change.